src/Microsoft.Azure.SignalR.Common/Utilities/ConnectionStringParseResult.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Azure.SignalR
+{
+    internal class ConnectionStringParseResult
```

Comment:
nit: `ConnectionDetail`, `ConnectionEntity`, `ParsedConnectionString`?

Reply:
updated to `ParsedConnectionString`